exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/ApiControllerIntegrationTest.java

```diff
@@ -762,21 +760,19 @@ void getActualConfiguration(VertxTestContext context) {
 
   @Test
   void getTime(VertxTestContext context) {
-    ZonedDateTime time = ZonedDateTime.of(2018, 1, 1, 0, 0, 0, 0, ZoneOffset.UTC);
+    TimeDto time = new TimeDto(ZonedDateTime.of(2018, 1, 1, 0, 0, 0, 0, ZoneOffset.UTC));
     when(qaService.getTime()).thenReturn(Optional.of(time));
 
-    String expectedStringTime = convertZdtToString(time);
-
     get(TIME_PATH)
         .send(context.succeeding(response -> context.verify(() -> {
           assertThat(response.statusCode())
               .isEqualTo(HTTP_OK);
 
           String body = response.bodyAsString();
-          Object actualTime = JSON_SERIALIZER
-              .fromJson(body, new TypeToken<String>() {
+          TimeDto actualTime = JSON_SERIALIZER
+              .fromJson(body, new TypeToken<TimeDto>() {
```

Comment:
JFYI can be simplified to `.fromJson(body, TimeDto.class);`

Reply:
Yep, TypeTokens are a workaround for generic types.